fedora/changelog

```diff
@@ -1,53 +1,3 @@
-%global srcname ogr
-
-Name:           python-%{srcname}
-Version:        0.11.1
-Release:        1%{?dist}
-Summary:        One API for multiple git forges
-
-License:        MIT
-URL:            https://github.com/packit-service/ogr
-Source0:        %{pypi_source}
-BuildArch:      noarch
-
-BuildRequires:  python3-devel
-BuildRequires:  python3dist(setuptools)
-BuildRequires:  python3dist(setuptools-scm)
-BuildRequires:  python3dist(setuptools-scm-git-archive)
-
-%description
-One Git library to Rule!
-
-%package -n     python3-%{srcname}
-Summary:        %{summary}
-%{?python_provide:%python_provide python3-%{srcname}}
-
-%description -n python3-%{srcname}
-One Git library to Rule!
-
-
-%prep
-%autosetup -n %{srcname}-%{version}
-# Remove bundled egg-info
-rm -rf %{srcname}.egg-info
-
-
-%build
-%py3_build
-
-
-%install
-%py3_install
-
-
-%files -n python3-%{srcname}
-%license LICENSE
-%doc README.md
-%{python3_sitelib}/%{srcname}
-%{python3_sitelib}/%{srcname}-%{version}-py?.?.egg-info
-
-
-%changelog
 * Wed Apr 01 2020 Jan Sakalos <sakalosj@gmail.com> - 0.11.1-1
 - patch release: 0.11.1
```

Comment:
Do we need this file? Cannot it use only git log?

Reply:
rpmautospec doesn't process git log entriess past the latest build.

You are asking here to drop all the changelog entries which is fine to do in upstream, but terrible to do in Fedora. Hence we could not include the file here and have it downstream only, which makes sense to me.

Reply:
OK, I get it now. I misunderstood that part from their docs.